.gitignore

```diff
@@ -36,3 +36,4 @@ pickle-email-*.html
 .env.development
 .env.test
 .env.local
+docker-compose.{env,yml}
```

Comment:
Just because I'm curious, what version of Git do you have and what OS do you run?